src/pid.js

```diff
@@ -0,0 +1,38 @@
+const path = require('path')
+const fs = require('fs')
+const terminate = require('terminate')
+const logger = require('./logger')
+
+class Pid {
+  constructor (userDataPath) {
+    this.loc = path.join(userDataPath, 'pid')
+  }
+
+  remove () {
+    return fs.unlinkSync(this.loc)
+  }
+
+  exists () {
+    return fs.existsSync(this.loc)
```

Comment:
I figure it's OK to block because if there is a pid we don't want to do anything until we delete it and restart the process

Reply:
+1